vg/font.go

```diff
@@ -37,23 +38,24 @@ var (
 	// Fonts that are not keys of this map are not supported.
 	FontMap = map[string]string{
 
-		// At the moment, we use fonts from GNU's freefont
-		// project.
+		// At the moment, we use fonts from RedHat's LIBERATION
```

Comment:
Might as well remove the "At the moment," clause. Since we can embed these, I would consider it a permanent solution.